JBoss Editorial for 25/02/2021 (#49)
jbossorg/jbossorg.github.io: src/content/posts/weekly-2021-02-25.adoc

= This Week in JBoss - 25 February 2021
Romain Pelisse
2021-25-11
:tags: quarkus, wildfly, kogito, openshift, bpm, microprofile, resteasy, keycloak, infinispan

Welcome to another installment of our JBoss Editorial! Today, we'll focus on Kogito and the upcoming major release of Wildfly. Enjoy!

== Wildfly 23 Beta: Microprofile

As the release of the next major version of Wildlfy is coming up soon, Brian Stanberry took the time to write up a brief article on the https://www.wildfly.org//news/2021/02/24/WildFly23-Beta-Released/[progress with MicroProfile]. If you have interested in playing with https://github.com/eclipse/microprofile/releases/tag/4.0[MicroProfile Platform 4.0] or see tech preview support for https://github.com/eclipse/microprofile-reactive-messaging/releases/tag/1.0[MicroProfile Reactive Messaging 1.0], go check out https://download.jboss.org/wildfly/23.0.0.Beta1/wildfly-23.0.0.Beta1.zip[Wildfly 23 Beta1] and let us know what you think!

== Kogito

> If you are already a Kogito users or if you are interested in exploring this software, there were a few articles released in the past two weeks that you might find interesting. The first one if on https://blog.kie.org/2021/02/kogito-notifications-api.html[Kogito Notification API] which can be followed nicely by the second one on https://blog.kie.org/2021/02/event-driven-decisions-with-kogito.html[Event-driven decisions with Kogito]

Reviewer: ```suggestion
If you are already a Kogito user or if you are interested in exploring this software, there were a few articles released in the past two weeks that you might find interesting. The first one is on https://blog.kie.org/2021/02/kogito-notifications-api.html[Kogito Notification API] which is followed nicely by the second one on https://blog.kie.org/2021/02/event-driven-decisions-with-kogito.html[Event-driven decisions with Kogito]
```

And if you have not yet looked into Quarkus, a new tutorial has been released: https://developers.redhat.com/blog/2021/02/11/enhancing-the-development-loop-with-quarkus-remote-development/[Enhancing the development loop with Quarkus remote development], so it's the perfect opportunity to explore the framework potential and test it over OpenShift.

== Evangelist's Corner

As always, our very own Eric Schabell has provided the community with quite a lot of content in the last two weeks. First, you have a nice https://www.schabell.org/2021/02/business-optimisation-an-introduction.html[introduction to Business optimization architecture], which can be followed by this other piece on https://www.schabell.org/2021/02/business-optimisation-common-architectural-elements.html[Business optimization architecture - Common architectural elements].

> Of course, Eric has not forgotten the famous OpenShift and delivered a pretty nifty https://www.schabell.org/2021/03/refcard-getting-started-with-openshift.html[RefCard - Getting started with OpenShift published] for DZone. Go grab it!

Reviewer: ```suggestion
Of course, Eric has not forgotten the famous OpenShift and delivered a pretty nifty https://www.schabell.org/2021/03/refcard-getting-started-with-openshift.html[RefCard - Getting started with OpenShift] published for DZone. Go grab it!
```
